MenuApplets/ResourceGraph/main.cpp

```diff
@@ -104,7 +104,7 @@ class GraphWidget final : public GUI::Frame {
         for (auto value : m_history) {
             painter.draw_line(
                 { rect.x() + i, rect.bottom() },
-                { rect.x() + i, (int)(rect.height() - (value * (float)rect.height())) },
+                { rect.x() + i, (int)(rect.top() + ((1 - value) * rect.height())) },
```

Comment:
An alternative would be

```cpp
(int)(rect.top() + rect.height() - (value * (float)rect.height()))
```

Reply:
Should we `round()` before casting to int too?

Reply:
Good point, will add that :)

Reply:
Yes rounding will make it work better:
```c++
rect.top() + (int)(round((float)rect.height() - (value * (float)rect.height())))
```
looks better to me.